library/markdown.php

```diff
@@ -6,14 +6,13 @@
  * @brief Parser for Markdown files
  */
 
-require_once "library/php-markdown/Michelf/MarkdownExtra.inc.php";
 use \Michelf\MarkdownExtra;
 
 /**
```

Comment:
Please remove/relocate this function as well. This was some wrapper from a time where no one seemed to understand autoloader and such stuff.

This file doesn't belong under /library since it is one of our functions.

Reply:
Not sure where to move it for now, I don't think we have agreed on anything, `Friendica\Content`?

Reply:
I think it belongs to the same place the bbcode parser and all other parsers belong to. I would like to have them all in a separate folder.